### DISCUSSION TITLE
about a IPA ad hoc.

### DISCUSSION THREAD

**Discussion by kanmurimori** [psb1558/Junicode-font#159] [Show and tell]
![image](https://user-images.githubusercontent.com/20810895/220369814-bbbc90d8-8457-498a-9b24-60c11205b352.png)
the ( on beta like u̟͡β̞ is in a not true position.

**Comment by psb1558** [comment]
This is a vertical kerning issue: the tie is raised when the letter before or behind is tall. I've added beta and the IPA-compatible variant of beta to the list of tall letters, and it will be correct whenever I generate a new version of Junicode.

**Comment by kanmurimori** [comment]
![image](https://user-images.githubusercontent.com/20810895/221396668-79fef80d-4582-43f4-92c7-85eec535f2b4.png)
t͡θ @psb1558

**Reply by psb1558** [reply]
Got it.
